service/src/main/java/com/commercetools/Main.java

```diff
@@ -0,0 +1,44 @@
+package com.commercetools;
+
+import com.commercetools.pspadapter.payone.*;
+import com.commercetools.pspadapter.payone.config.PropertyProvider;
+import com.commercetools.pspadapter.payone.config.ServiceConfig;
+import org.quartz.CronScheduleBuilder;
+import org.quartz.SchedulerException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+
+import static java.lang.String.format;
+
+public class Main {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) throws SchedulerException, MalformedURLException {
+
+        final PropertyProvider propertyProvider = new PropertyProvider();
+        final ServiceConfig serviceConfig = new ServiceConfig(propertyProvider);
+
+        final IntegrationService integrationService = ServiceFactory.createIntegrationService(propertyProvider, serviceConfig);
+        integrationService.start();
+
+        ScheduledJobFactory scheduledJobFactory = new ScheduledJobFactory();
+
+        scheduledJobFactory.setAllScheduledItemsStartedListener(() ->
+                LOG.info(format("%n%1$s%nPayone Integration Service is STARTED%n%1$s",
+                "============================================================"))
+        );
+
+        scheduledJobFactory.createScheduledJob(
+                CronScheduleBuilder.cronSchedule(serviceConfig.getScheduledJobCronForShortTimeFramePoll()),
+                ScheduledJobShortTimeframe.class,
+                integrationService);
+
+        scheduledJobFactory.createScheduledJob(
+                CronScheduleBuilder.cronSchedule(serviceConfig.getScheduledJobCronForLongTimeFramePoll()),
+                ScheduledJobLongTimeframe.class,
+                integrationService);
+    }
```

Comment:
the comments are added in [`ScheduledJobLongTimeframe`](https://github.com/commercetools/commercetools-payone-integration/blob/d4a0c651f4d4e3f5bb59df8f367e43e4d5108746/service/src/main/java/com/commercetools/pspadapter/payone/ScheduledJobLongTimeframe.java#L8-L8) and [`ScheduledJobShortTimeframe`](https://github.com/commercetools/commercetools-payone-integration/blob/d4a0c651f4d4e3f5bb59df8f367e43e4d5108746/service/src/main/java/com/commercetools/pspadapter/payone/ScheduledJobShortTimeframe.java#L8-L8) classes.

Anyway: these feature looks redundant now and likely will be removed soon.